config.template

```diff
@@ -121,6 +121,18 @@ areaNH = IceArea_timeseries/iceAreaNH_climo.nc
 areaSH = IceArea_timeseries/iceAreaSH_climo.nc
 volNH = PIOMAS/PIOMASvolume_monthly_climo.nc
 volSH = none
+concNASATeamWinNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concNASATeamSumNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concNASATeamWinSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concNASATeamSumSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_jja.interp0.5x0.5.nc
+concBootstrpWinNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concBootstrpSumNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concBootstrpWinSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concBootstrpSumSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_jja.interp0.5x0.5.nc
+thickonNH = ICESat/ICESat_gridded_mean_thickness_NH_on.interp0.5x0.5.nc
+thickfmNH = ICESat/ICESat_gridded_mean_thickness_NH_fm.interp0.5x0.5.nc
+thickonSH = ICESat/ICESat_gridded_mean_thickness_SH_on.interp0.5x0.5.nc
+thickfmSH = ICESat/ICESat_gridded_mean_thickness_SH_fm.interp0.5x0.5.nc
 
```

Comment:
not at all, on the contrary @vanroekel!
About 1: you are right, the variable names are a bit cryptic, I will change them. I was following the template of the names above for the timeseries plot.
About 2: I am OK with giving the user the ability to change the obs files completely, because, at least for now, these files go hand in hand with the remap file that we also specify. All of these have to be consistent with each other, and the user needs to know that.
Now, things are likely to change in the future, when @xylar will put in the Interpolation module, but until then this seems the correct approach to me.
Do you agree?